controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -2193,6 +2196,27 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
       });
     },
 
+    getOrganizationWebhookMeta: (req, ctx) => {
+      const logger = opts.logger.child({
+        service: ctx.service.typeName,
+        method: ctx.method.name,
+      });
+
+      return handleError<PlainMessage<GetOrganizationWebhookMetaResponse>>(logger, async () => {
+        const authContext = await opts.authenticator.authenticate(ctx.requestHeader);
+        const orgRepo = new OrganizationRepository(opts.db);
+
+        const meta = await orgRepo.getWebhookMeta(req.id, authContext.organizationId);
+
+        return {
+          response: {
+            code: EnumStatusCode.OK,
+          },
+          meta: JSON.stringify(meta),
```

Comment:
Please use proper types.

Reply:
Proto doesnt give me the flexibility with json as typescript. (Related [comment](https://github.com/wundergraph/cosmo/pull/149#discussion_r1346456711)). I do use the types on both ends

Reply:
Why is it so dynamic? I saw we a fixed type for that in shared.

Reply:
Yeah but thats in typescript and I do use it. Maybe I am not familiar with sending json without stringify

Reply:
Fixed